net/dns-over-https/Makefile

```diff
@@ -0,0 +1,56 @@
+# SPDX-License-Identifier: GPL-2.0-only
+#
+# Copyright (C) 2021 Martin Schneider <martschneider@google.com>
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=dns-over-https
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/m13253/dns-over-https
+PKG_SOURCE_DATE:=2021-09-29
+PKG_SOURCE_VERSION:=b338c7ae52bed0d1a5e85e30c76d023efb46580d
```

Comment:
Suggest to use a release tag unless there is a reason to package a specific commit.

Can also consider using codeload.